### ISSUE TITLE
cli tool

### ISSUE THREAD

**Issue by andorsk** [TBD54566975/ssi-sdk#150]
build a cli tool to support various operations and. verifications. Inspiration can be seen here: https://www.youtube.com/watch?v=AVnCVzW0rkI

**Comment by decentralgabe** [comment]
I like the idea. Guessing this should be a new project that uses the SDK?

**Comment by andorsk** [comment]
@decentralgabe maybe? I'm a little worried about run-away repo management. There's the `cmd` folder, that we could use at the beginning and if it gets to be a large project move it over?

**Comment by decentralgabe** [comment]
Yeah @andorsk agree, I think keeping it in a cmd directory here is a good first step.

**Comment by andorsk** [comment]
started a PR here: https://github.com/TBD54566975/ssi-sdk/pull/183

**Comment by decentralgabe** [comment]
Check out https://lists.w3.org/Archives/Public/public-credentials/2022Sep/0215.html
